Makefile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
one question, should here be `kubernetes authors`?

Reply:
Not sure. What would you suggest? I put kubernetes because: this is a very specific to kubernetes code, this work is also based on kubernetes provided plugin example and I am part of kubernetes org, but I do not mind to change it if a good reason is provided.